src/Kernel32.Tests/storebanned/Kernel32Facts.cs

```diff
@@ -1130,6 +1130,43 @@ unsafe uint GetProcessIdThreadProc(void* data)
         GC.KeepAlive(threadStartRoutine); // Make sure that the delegate stays alive until the test is done
     }
 
+    [Fact]
+    public unsafe void DeviceIOControl_Works()
+    {
+        const uint IOCTL_DISK_GET_DRIVE_GEOMETRY = 0x070000;
```

Comment:
Common I/O codes are defined in https://docs.microsoft.com/en-us/windows/win32/api/winioctl/ , might make sense to add them to dotnet/pinvoke (as a separate effort)